magma/syntax/inline_combinational.py

```diff
@@ -0,0 +1,95 @@
+import ast
+from typing import MutableMapping, Optional
+
+import astor
+
+from ast_tools.stack import SymbolTable
+from ast_tools.passes import (apply_ast_passes, Pass, PASS_ARGS_T, ssa,
+                              if_to_phi, bool_to_bit)
+from ast_tools.common import gen_free_prefix
+from ast_tools.immutable_ast import immutable, mutable
+
+from magma.syntax.combinational2 import combinational2
+from magma.bit import Bit
+from magma.wire import wire
+
+
+class _ToCombinationalRewriter(ast.NodeTransformer):
+    def __init__(self, free_prefix, target_map):
+        self.prefix = free_prefix
+        self.target_map = target_map
+        self.name_count = 0
+
+    def _gen_new_name(self):
+        new_name = f"{self.prefix}{self.name_count}"
+        self.name_count += 1
+        return new_name
+
+    def visit_AugAssign(self, node):
+        if not isinstance(node.op, ast.MatMult):
+            return
```

Comment:
This will delete aug assigns that are not matmul.  Probably want to raise an error because ssa doesn't transform them or minimally return node.

Reply:
Woops, yea should just return for now I think